src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -141,6 +158,36 @@ which we divide into several categories:
 
 #### Radiochemistry
 
+{{ MACROS___make_metadata_table(
+   {
+      "TracerName": "REQUIRED",
+      "TracerRadionuclide": "REQUIRED",
+      "InjectedRadioactivity": "REQUIRED",
+      "InjectedRadioactivityUnits": "REQUIRED",
+      "InjectedMass": "REQUIRED",
+      "InjectedMassUnits": "REQUIRED",
+      "SpecificRadioactivity": "REQUIRED",
+      "SpecificRadioactivityUnits": "REQUIRED",
+      "ModeOfAdministration": "REQUIRED",
```

Comment:
in the ephys specs, these huge tables are separated between required and recommended fields, see e.g., https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/03-electroencephalography.html#sidecar-json-_eegjson

I think that'd be reasonable to do here as well.

Reply:
Can we push this one to another PR? I would prefer to have the PET folks weigh in on splitting tables, since it was a point of discussion in the BEP IIRC, and I don't want to introduce any potentially lengthy conversations to this PR.

Reply:
yes, no worries.